python/sglang/srt/layers/moe/fused_moe_triton/layer.py

```diff
@@ -28,6 +33,8 @@
 
 import logging
 
+from sgl_kernel.ops._kernels import convert_weight_packed, fused_experts_cpu
```

Comment:
if sglang csrc are not compiled with cpu support, this will cause error, we should avoid directly import sgl_kernel.ops._kernels.xxx here

just call prepack_weight_if_needed from this file